src/main/java/uk/gov/hmcts/darts/arm/component/impl/ArmResponseFilesProcessSingleElementImpl.java

```diff
@@ -222,6 +229,71 @@ private void processUploadFile(String armInputUploadFilename, List<String> respo
         }
     }
 
+    private void getAndLogCreateRecordFile(ExternalObjectDirectoryEntity externalObjectDirectory, String createRecordFilename) {
+        if (nonNull(createRecordFilename)) {
+            Path jsonPath = null;
+            try {
+                CreateRecordFilenameProcessor createRecordFilenameProcessor = new CreateRecordFilenameProcessor(createRecordFilename);
+                BinaryData createRecordFileBinary = armDataManagementApi.getBlobData(createRecordFilenameProcessor.getCreateRecordFilenameAndPath());
```

Comment:
Is this approach safe for large files (if large files are possible/relevant to this use case?)

Reply:
these are response files which are small json files so size wont be an issue. There is going to be a ticket to change all the ARM stuff from BinaryData but its not been played yet.